addons/web/static/tests/webclient/actions/concurrency_tests.js

```diff
@@ -695,6 +695,32 @@ QUnit.module("ActionManager", (hooks) => {
         assert.containsOnce(webClient, ".o_form_view");
     });
 
+    QUnit.test(
+        "dialog will only open once for two rapid actions with the target new",
+        async function (assert) {
+            assert.expect(3)
+            const def = makeDeferred();
+            const mockRPC = async (route, args) => {
+                if (args.method === "onchange") {
+                   return def;
+                }
+            };
+
+            const webClient = await createWebClient({ serverData, mockRPC });
+            doAction(webClient, 5);
+            await nextTick();
+            assert.containsNone(webClient, ".o_form_view");
+
+            doAction(webClient, 25);
```

Comment:
I would execute twice the same action (s.t. we know both are target="new")
```js
doAction(webClient, 5);
```

Reply:
@aab-odoo Done. Thank You :slightly_smiling_face: